src/applications/personalization/profile/tests/components/alerts/Profile.data-unavailable-error.unit.spec.js

```diff
@@ -7,11 +7,11 @@ import { setupServer } from 'msw/node';
 
 import { resetFetch } from 'platform/testing/unit/helpers';
 
-import * as mocks from '../../msw-mocks';
-import { renderWithProfileReducers as render } from '../unit-test-helpers';
+import * as mocks from '../../../msw-mocks';
```

Comment:
...and if we added that Babel alias this import would become `import * as mocks from 'profile/msw-mocks'`;